### PR TITLE
@kanaabe: close onboarding a/b test and send all users to the new flow

### PR DESCRIPTION
**Pull request by xtina-starr**
This is pretty straight forward, but want to make sure I don't miss anything.

### CODE REVIEW

File: src/desktop/apps/personalize/routes.js [hunk 1]
```diff
@@ -1,25 +1,10 @@
...
-          res.locals.sd.CURRENT_USER
-        )
-        res.render('template')
-      },
-    })
-  } else {
-    newOnboarding(req, res, next)
-  }
-}
+let renderLayout = _renderLayout
```
Comment (kanaabe) [inline src/desktop/apps/personalize/routes.js]: What's this for?

File: src/desktop/apps/personalize/test/routes.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  renderLayout = sinon.stub()
+  rewire.__set__('renderLayout', renderLayout)
+
+  it('renders the personalize app', () => {
+    index(req, res, next).then(() => {
+      renderLayout.args[0][0].data.title.should.eql('Personalize | Artsy')
+      renderLayout.args[0][0].locals.assetPackage.should.eql('onboarding')
+    })
+  })
+})
```
Comment (kanaabe) [inline src/desktop/apps/personalize/test/routes.js]: Thanks for the test!

File: src/desktop/components/split_test/running_tests.coffee [hunk 3]
```diff
@@ -16,13 +16,6 @@
 # module.exports = {}
 
 module.exports = {
-  onboarding_test:
-    key: 'onboarding_test'
-    outcomes:
-      control: 50
-      experiment: 50
-    control_group: 'control'
-    edge: 'experiment'
```
Comment (kanaabe) [inline src/desktop/components/split_test/running_tests.coffee]: 🎉

**Comment by kanaabe** [comment]
Ohh we forgot mobile here. Starting point is: https://github.com/artsy/force/blob/master/src/mobile/apps/personalize/index.coffee. And actually, we can just take out this app and all its associated assets because on mobile users will fall through the middleware and end up on responsive desktop. 

Besides that and one small q I had on `_renderLayout`, this looks great! So excited for that code diff too. Congrats on a successful test 👏

File: src/desktop/apps/personalize/routes.js (hunk 1 above)
Comment (xtina-starr) [reply src/desktop/apps/personalize/routes.js]: @kanaabe in the test set up `renderLayout` returns an **undefined error** when it get's passed to the `rewire` function. I saw this renaming wizardry in the [auctions app](https://github.com/artsy/force/blob/master/src/desktop/apps/auction/routes.js#L18-L22) and it seems to make `renderLayout` available.

**Comment by xtina-starr** [comment]
Ah good catch regarding mobile!

**Pull request merged**